docs/styleguide.rst

```diff
@@ -0,0 +1,24 @@
+.. _styleguide:
```

Comment:
Note that to update the docs, you'll have to manually rebuild the docs via the readthedocs portal.
It hasn't yet been officially connected to this repository to auto-build.

Reply:
That's okay, it's not really complete yet anyways